janitor/functions.py

```diff
@@ -2318,6 +2319,8 @@ def row_to_names(
         Defaults to False.
     :param remove_rows_above: Whether the rows above the selected row should
         be removed from the DataFrame. Defaults to False.
+    :reset_index : Whether the index should be reset on the returning DataFrame.
+    	Defaults to True.
```

Comment:
Just a minor nitpick suggestion that I will commit straight away.

```suggestion
    :reset_index : Whether the index should be reset on the returning DataFrame.
        Defaults to True.
```